sdl_android/src/main/java/com/smartdevicelink/transport/MultiplexBluetoothTransport.java

```diff
@@ -280,21 +277,21 @@ public synchronized void connected(BluetoothSocket socket, BluetoothDevice devic
         Bundle bundle = new Bundle();
```

Comment:
We should handle the case of device being null here when calling device.getName().